### PR TITLE
Adds Process for technical design docs

### PR DESCRIPTION
**Pull request by seanmalbert**

### CODE REVIEW

File: docs/governance-and-management/governance/governance-processes.md [hunk 1]
```diff
@@ -66,3 +66,30 @@ Types of technical design decisions that require approval:
...
+
+This section outlines the process for features that require a technical design document. The purpose of a design document is to draft and approve major technical design decisions to ensure the quality, reliability, and maintainability of CiviForm for all its users.
+
+#### Before You Start
+
+The following should be true:
+1. There is an open issue in [GitHub](https://github.com/civiform/civiform/issues)
+2. There is an approved PRD in the [public folder)(https://drive.google.com/drive/u/0/folders/1gqw3VSK-lMPgRfcS_ZEhF3C_51We0eGz)
+3. If the issue has a `UX` label, approved UX Mocks are added to the [public folder](https://drive.google.com/drive/u/0/folders/1zY6OnQYH7Xo5ebPctTJtd5QRsHBAI5vF)
+4. A `Ready for TDD` label is added to the issue and visible in [this list](https://github.com/civiform/civiform/issues?q=is%3Aissue+is%3Aopen+label%3AReady+for+TDD)
```
Comment (bion) [inline docs/governance-and-management/governance/governance-processes.md:79]: question (blocking): does this label exist yet?

Comment (bion) [inline docs/governance-and-management/governance/governance-processes.md:79]: question (blocking): it looks like the label is unquoted in the search query which may not work as expected: 
image

File: docs/governance-and-management/governance/governance-processes.md [hunk 2]
```diff
@@ -66,3 +66,30 @@ Types of technical design decisions that require approval:
...
+
+During the `CiviForm eng weekly` meeting on Mondays. We review the issues with the `Ready for TDD` label and assign an author, approvers and reviewers. Assignment can be during the week either in a daily status meeting or by discussing in the [engineering slack channel](https://civiform.slack.com/archives/C01QKN2UNMA).
```
Comment (bion) [inline docs/governance-and-management/governance/governance-processes.md:81]:
```suggestion
During the `CiviForm eng weekly` meeting on Mondays we review the issues with the `Ready for TDD` label and assign an author, approvers and reviewers. Assignment can be during the week either in a daily status meeting or by discussing in the [engineering slack channel](https://civiform.slack.com/archives/C01QKN2UNMA).
```

Comment (bion) [reply docs/governance-and-management/governance/governance-processes.md:81]: nit: usually like to reserve the backticks for code-related things vs names like the name of the meeting

Comment (bion) [reply docs/governance-and-management/governance/governance-processes.md:81]: issue (blocking): it's not clear from these sentences the difference between things assigned at the weekly eng meeting vs the daily status meeting and slack discussions

File: docs/governance-and-management/governance/governance-processes.md [hunk 3]
```diff
@@ -66,3 +66,30 @@ Types of technical design decisions that require approval:
...
+
+#### Working with the Doc
```
Comment (bion) [inline docs/governance-and-management/governance/governance-processes.md:83]: suggestion (non-blocking): recommend adding some discussion of how to choose approvers and the difference between approvers and reviewers

**Review by bion — CHANGES_REQUESTED** [review]

File: docs/governance-and-management/governance/governance-processes.md (hunk 1 above)
Comment (seanmalbert) [reply docs/governance-and-management/governance/governance-processes.md:79]: No, I was holding off on creating the label until this is approved.

File: docs/governance-and-management/governance/governance-processes.md (hunk 2 above)
Comment (seanmalbert) [reply docs/governance-and-management/governance/governance-processes.md:81]: There isn't any difference, other than I'm suggesting that there will be dedicated time in the weekly eng meeting and others can be ad hoc, if we don't want to wait for the next meeting. I'm also fine with saying that it only happens on the Monday meetings, but that feels like it's unnecessarily blocking. I can make that more clear though.

File: docs/governance-and-management/governance/governance-processes.md (hunk 1 above)
Comment (seanmalbert) [reply docs/governance-and-management/governance/governance-processes.md]: Added the label.

Comment (seanmalbert) [reply docs/governance-and-management/governance/governance-processes.md]: Updated link

File: docs/governance-and-management/governance/governance-processes.md (hunk 2 above)
Comment (seanmalbert) [reply docs/governance-and-management/governance/governance-processes.md]: Updated the language. I imagine that this section will be iterated on as it gets put into practice.

**Review by bion — APPROVED** [review]

**Pull request merged**